package.json

```diff
@@ -56,8 +56,8 @@
     "inter-ui": "~3.19.3",
     "lodash": "~4.17.21",
     "lodash-inflection": "~1.5.0",
-    "mobx": "~6.8.0",
-    "mobx-react-lite": "~3.4.0",
+    "mobx": "~6.9.1",
+    "mobx-react-lite": "~3.4.3",
```

Comment:
Note that this lib has moved to 4.x releases - worth looking at those updates in this branch?